install_config/upgrading/blue_green_deployments.adoc

```diff
@@ -15,26 +15,32 @@ toc::[]
 
 [NOTE]
 ====
-This topic serves as an alternative node upgrade method to the approach in
-xref:../../install_config/upgrading/manual_upgrades.adoc#upgrading-nodes[Manual In-place Upgrades].
+This topic serves as an alternative approach for node upgrades to the in-place
+upgrade method.
 ====
 
-Blue-green deployments are a proven approach to reducing downtime caused while
-upgrading an environment. This is done by creating a parallel environment on
-which the new deployment can be installed. If a problem is detected, and after
-the new deployment is verified, traffic can be switched over with the option to
-rollback.
+The
+xref:../../install_config/upgrading/blue_green_deployments.adoc#upgrading-blue-green-deployments[blue-green deployment] upgrade method follows a similar flow to the in-place method:
+masters and etcd servers are still upgraded first, however a parallel
+environment is created for new nodes instead of upgrading them in-place.
```

Comment:
This last instance of in-place should be "in place" (not "in-place") since it's not modifying a noun, like "in-place method" earlier in the paragraph